containers/base/build.sh

```diff
@@ -20,16 +20,18 @@
 
 # Build the docker image
 if [ -n "$1" ]; then
-  src_pydatalab="$1"
+  src_pydatalab=$(realpath "$1")
 
   # Append trailing / if needed.
   if [ "${src_pydatalab: -1}" != "/" ]; then
```

Comment:
A better fix is of course to make the code accept the path without the slash, but for the purposes of this PR, removing the if check is enough.